quinn-h3/src/client.rs

```diff
@@ -528,13 +527,21 @@ where
         conn: ConnectionRef,
         request: Request<B>,
     ) -> Self {
+        if conn.h3.lock().unwrap().inner.is_closing() {
```

Comment:
Might be nice to add a test for this? I think that would be easy to do from your existing `GoAway` test?

Reply:
I got it differentiated.